DirectEmployers/MyJobs
# [PD 2809] replacing calendar

This task was to replace the bootstrap daterangpicker with one that we currently have.

To Test:
Pull down the repository. You can click on the button with the Date in the header and see the calendar come down. The first part is the quick range selections which will trigger an API call and you can see the correct data going into the table in terms of filtering. You can also see the response coming back in the network Tab. If you click custom range inside of the dropdown it will pop out the calendar which has NOT been linked as of yet. This is be a different PR to set it with two calendars as a date range selection.
